CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/NoResult/NoResultScreenViewController.swift

```diff
@@ -327,7 +314,7 @@ final public class NoResultScreenViewController: UIViewController {
 
     private func configureButtons() {
         configureButtonsColors()
-        addBlurEffect(button: enterButton, cornerRadius: 14)
+        enterButton.addBlurEffect(cornerRadius: 14)
```

Comment:
[Suggestion] it makes sense to reuse _outlineButtonCornerRadius_ from _GiniConfiguration_.